packages/govern-server/src/core/thegraph-queries/registry.ts

```diff
@@ -13,20 +13,72 @@ const RegistryBase = gql`
   }
 `
 
+const RegistryEntryComplete = gql`
+  fragment RegistryEntryComplete on RegistryEntry {
+    id
+    name
+    queue {
+      id
+      address
+      config {
+        executionDelay
+        scheduleDeposit {
+          id
+          token
+          amount
+        }
+        challengeDeposit {
+          id
+          token
+          amount
+        }
+        resolver
+        rules
+      }
+      queued {
```

Comment:
would rename ``queued`` to ``queue``  and isn't it possible to use ``fragments`` here? will lower the maintenance burden..

Reply:
That’s how it used to be, but we renamed it to avoid having `queue.queue`. We now only use `queue` to refer to a `GovernQueue`, and `queued` when it is a list of `Containers`.